### PR TITLE
feat: Added Documentation Page

### PR DESCRIPTION
**Pull request by Yashagrahari07**
Added Docs page and  Documentation is meticulously crafted to provide developers with a seamless and intuitive experience as they explore, integrate, and leverage the capabilities of the Velvet UI's components.
 
### ScreenShots

- **Desktop View**
laptop

- **Tablet View**
tablet

- **Mobile View**
mobile

### CODE REVIEW

File: css/avatar.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+@import 'variables.css';
+
+.avt-img {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-around;
+    gap: 50px;
```
Comment (NishantKrTiwari) [inline css/avatar.css:8]: use rems

File: css/buttons.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
+@import 'variables.css';
+
+.btn {
+    font-size: 1.1rem;
+    font-family: var(--font-3);
+    padding: 10px;
+    border-radius: 7px;
+    border: none;
+    box-shadow: 2px 2px 20px #00000033;
```
Comment (NishantKrTiwari) [inline css/buttons.css:9]: use css variables for color

File: css/buttons.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+.btn-primary {
+    background-color: var(--pri-color);
+    color: white;
+}
+
+.btn-secondary {
+    background-color: var(--sec-color);
+    color: white;
```
Comment (NishantKrTiwari) [inline css/buttons.css:21]: here as well

File: css/cards.css [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    background-color: #fff;
+    border-radius: 12px;
+  }
+
+  .back {
+    transform: rotateY(180deg);
+  }
+
+  .profile {
+    width: 150px;
```
Comment (NishantKrTiwari) [inline css/cards.css:99]: use rems for bigger units

File: css/docs.css [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+.sidebar::-webkit-scrollbar {  /* ScrollBar width */
+  width: 10px;
+}
+
+.sidebar::-webkit-scrollbar-thumb {  /* ScrollBar thumb Styling */
+  border-radius: 10px;
+  background-color: var(--light-color);
+}
+
+.sidebar::-webkit-scrollbar-track {  /* ScrollBar track Styling */
```
Comment (NishantKrTiwari) [inline css/docs.css:54]: why use webkit ?

**Review by NishantKrTiwari — COMMENTED** [review]
you are are supposed to create seperate PR for each feature, not like this

File: css/docs.css (hunk 5 above)
Comment (Yashagrahari07) [reply css/docs.css:54]: for styling the track of scroll bar

File: css/avatar.css (hunk 1 above)
Comment (Yashagrahari07) [reply css/avatar.css:8]: ok

**Pull request merged**